schema/codelists/open/quantity_kind.csv

```diff
@@ -0,0 +1,5 @@
+Code,Title
+area,Area
+count,Count
+currency,Currency
+length,Length
```

Comment:
Add descriptions

```suggestion
Code,Title,Description
area,Area,The two-dimensional size of the part of a surface.
count,Count,A count of items.
currency,Currency,A financial value.
length,Length,A measure of distance.
```

Reply:
agree to add them but it's Definition not Description to match all of the other codelists

Reply:
Sounds good!